rfc-029-source-code-should-be-open-source-by-default.md

```diff
@@ -0,0 +1,144 @@
+people to do what they wish with the code and removes any potential
+legal issues for us. The other licence that came up is some version of
+the [GPL](https://choosealicense.com/licenses/gpl-3.0/), but even if
+it were something we wished to use, we wouldn’t have the resources to
+monitor for and pursue license violations.
+
+## Proposal
+
+All new GitHub respositories in the dxw organisation SHOULD be public,
+unless they directly relate to a client project. When these
+repositories are public, they SHOULD contain a copy of the [MIT
+License](https://choosealicense.com/licenses/mit/) in a `LICENSE` file
+in the repository’s root.
+
+We MAY allow a non-client repository to be private if it contains:
+
+- secrets such as keys—although this is bad practice and should not
+  have happened in the first place
+- personal information
+- commercially sensitive information
+
+If we choose to make such a repository private, it MUST contain a
+`WHY_PRIVATE.md` document explaining why it’s private, with reference
+to one of these reasons. This is useful because:
+
+- it allows us to one day revisit the decision to make the repository
+  private, without relying on team members’ memories
+- we can set up automated checks to ensure that all private
+  repositories contain this justification
+
+When starting a new client project, we SHOULD ask the client whether
+we can make the repository public. We SHOULD explain the benefits of
+making it public.  Where relevant, we can refer to existing guidance
+such as the [Government Design
+Principles](https://www.gov.uk/guidance/government-design-principles#make-things-open-it-makes-things-better)
+or the [GOV.UK Service
+Standard](https://www.gov.uk/service-manual/service-standard/point-12-make-new-source-code-open).
+
+### Concerns
+
+When we write code for internal use, we don’t always apply the same
+standards which we would for a client project. For example, we have
+some repositories which are proofs of concept, or constrained by
+time. If somebody outside of dxw sees one of these repositories
+without understanding the context in which it was made, it might give
+a negative impression of the quality of our work.
+
+We SHOULD reduce this risk by pinning some repositories which we’re
+proud of, so that they appear at the top of dxw’s GitHub page.
+
+We already considered a similar risk of being misunderstood in [RFC
+28](rfc-028-adopt-a-code-of-conduct.md)—namely, that adding a code of
+conduct might suggest that we’re actively seeking community
+contributions. We said that we can mitigate this risk by using the
+`CONTRIBUTING.md` file to explain our likely response to
+contributions. And so, in the case of public repositories, we MAY also
+use a repository’s readme to explain the state of the project – for
+example, that it’s a proof of concept.
+
+Although one of the artifacts of our work is open source, sometimes
+useful software, our principles emphasise that we are *working in the
```

Comment:
I found this phrasing a little confusing. I kept reading the commas as clause breaks, rather than a list of adjectives.

Reply:
Definitely was confusing. I've changed it, thanks.